phpBB/styles/prosilver/theme/pm.css

```diff
@@ -0,0 +1,109 @@
+/* -------------------------------------------------------------- /*
+    Styles for the refactored PM module
+/* -------------------------------------------------------------- */
+
+@media (min-width: 768px) {
+	.pm {
+		background-color: #d3d3d3;
+		display: flex;
+	}
+}
+
+.pm-title {
+	white-space: nowrap;
+	text-overflow: ellipsis;
+	display: inline-block;
+	overflow: auto;
+	max-width: 200px;
+	margin-bottom: -6px;
+}
+
+.pm-checkbox {
+	font-size: 20px;
+	cursor: pointer;
+}
+
+.pm-list-container {
+	flex: 1;
+}
+
+.pm-meta {
+	color: rgba(0, 0, 0, 0.25);
+	opacity: 0;
+	justify-content: center;
+	align-items: center;
+	width: 20px;
+}
+
+.pm-viewer {
+	background-color: #d3d3d3;
+	flex: 2;
+	padding: 1.5rem;
+}
+
+.pm-list {
+	background-color: #edf4f7;
+	flex: 1 auto;
+	margin: 0;
+	padding: 0;
+	list-style-type: none;
+}
+
+.pm-authour {
+	white-space: nowrap;
+	display: block;
+	overflow: hidden;
+	width: 220px;
+}
+
+.pm-item {
+	font-size: 1rem;
+	font-weight: 400;
+	line-height: 1.5;
+	letter-spacing: 0.04em;
+	border-bottom: 1px solid rgba(0, 0, 0, 0.05);
+	position: relative;
+	display: flex;
+	align-items: center;
+	box-sizing: border-box;
+	height: 72px;
+	padding: 16px;
+}
+
+.pm-primary,
+.pm-secondary {
+	display: block;
+}
+
+.pm-primary {
+	line-height: 1.5;
+	flex-grow: 1;
+}
+
+.pm-secondary {
+	font-size: 13px;
+	font-weight: 400;
+	line-height: 1.25;
+	text-decoration: inherit;
+	letter-spacing: 0.04em;
+	text-transform: inherit;
+}
+
+.pm-secondary-item {
+	line-height: 1.2857142857;
+	padding-right: 1rem;
+}
+
+/*	colours and backgrounds  */
+
+.pm-unread .pm-title {
+	color: #d41142;
+}
+
+.fa-star {
+	color: #ff9500;
+}
+
+.fa-certificate {
```

Comment:
absolutely not should be `.pm-unread`

Reply:
but this changes the date color as well as they are in same tag

Reply:
pm-unread will cause both icon and date to change colour. Rather we could create another tag like pm-checkbox.

Reply:
No you have the pm-unread in the wrong place pls check the Costen and update your html